.cargo/config

```diff
@@ -0,0 +1,7 @@
+[alias]
+run-wasm = "run --release --package run-wasm --"
+
+[build]
+rustflags = [
+"--cfg=web_sys_unstable_apis"
```

Comment:
does this affect all of the builds, even unrelated to WASM?

Reply:
It is set for all builds but I cant see a reason the flag would affect non wasm targets.
And this is actually good because having all targets use the same rustflags prevents cargo from trashing our build cache.